FlowCal/transform.py

```diff
@@ -83,21 +83,34 @@ def transform(data, channels, transform_fxn, def_channels = None):
 
     return data_t
 
-def to_rfi(data, channels=None, max_range=None, amplification_type=None):
+def to_rfi(data,
+           channels=None,
+           amplification_type=None,
+           amplifier_gain=None,
+           max_range=None):
     """
     Transform flow cytometry data to Relative Fluorescence Units (RFI).
 
-    This function applies the following transformation:
+    If ``a[0]`` is different from zero, data has been taken using a log
```

Comment:
unclear what `a[0]` is referring to. I would think this should refer to the parameters listed above in the function signature.